x-pack/plugins/gis/public/_main.scss

```diff
@@ -43,14 +43,18 @@ map-listing, .gisListingPage {
 .gisWidgetOverlay {
   position: absolute;
   z-index: $euiZLevel1;
-  min-width: 17rem;
-  max-width: 24rem;
   top: $euiSizeM;
   right: $euiSizeM;
   bottom: $euiSizeM;
+  // left: $euiSizeM;
```

Comment:
I accidentally left this in, but I'll do a SASS cleanup soon-ish as a separate PR